src/controller/mainController.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 const controller = {
   home: (req, res) => {
-    return res.sendFile(path.resolve('./src/views/index.html'));
+    return res.sendFile(path.resolve('./public/views/index.html'));
```

Comment:
The public folder does not have `views`, it's only located in the `src`

Reply:
@TAKANOME-DEV In order for the links to be fixed I have to move /views to /public. I'm working on it now, but see my latest comment. Let me know what you think.